src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/GitFileManager.cs

```diff
@@ -184,6 +190,10 @@ public async Task<IEnumerable<DependencyDetail>> ParseVersionDetailsXmlAsync(str
             versionAttribute.Value = dependency.Version;
             newDependency.Attributes.Append(versionAttribute);
 
+            XmlAttribute pinnedAttribute = versionDetails.CreateAttribute("Pinned");
```

Comment:
I think we shouldn't create the attribute unless pinned is true.  Maybe a little too much clutter for something that will happen not all too often.